src/components/calculator-component.js

```diff
@@ -0,0 +1,11 @@
+function calculator(bodyId,screenId,inputId,buttonsGroup) {
+    return`
+    <div id="${bodyId}">
+        <div id="${screenId}">
```

Comment:
you don't need this!

Reply:
what do you mean? i let the input without a div? just inside the **"bodyId?"**

Reply:
why `bodyId` and not just `id`?

Reply:
the screenId i defined just as "screen" dynamically

Reply:
the calculator's `id` ok make sense! but the rest I don't see the benefit for them!

Reply:
i'm thinking about the person get the calculator and choose the id by themself, but this don't make sense
because the functions and the styles will run on these default id, right?